project_setup/scripts/terraform-to-secrets

```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/env python
+
+"""Parse Terraform state and create GitHub secrets.
+
+Secrets are create for IAM access keys, and specially tagged resources.
+
+For each IAM access key two secrets will be created:
+ - AWS_ACCESS_KEY_ID
+ - AWS_SECRET_ACCESS_KEY
+
+If there is only one access key, the secrets will be named as above.  If more than one
+key exists in the Terraform state, the key's name will be appended to the secret's name.
+ - AWS_ACCESS_KEY_ID_BUILDER
+ - AWS_SECRET_ACCESS_KEY_BUILDER
+
+Resources tagged with "GitHub_Secret_Name" and "GitHub_Secret_Terraform_Lookup" will
+have a single secret created based on the tag contents.  For example if a resource had
+these tags:
+ - GitHub_Secret_Name: BUILD_ROLE_TO_ASSUME
+ - GitHub_Secret_Terraform_Lookup: arn
+
+A secret would be generated with the name "BUILD_ROLE_TO_ASSUME" and the value of
+the resource's ARN.
+
+This command must be executed in the directory containing a .terraform state
```

Comment:
Should be "executed in the directory containing the" or "executed in a directory containing a", not mixed.